### PR TITLE
add changelog for bugfix 18164 prometheus metric register of walWriteSec

### PR DESCRIPTION
**Pull request by andyxning**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

change log for bugfix https://github.com/etcd-io/etcd/pull/18164.

/cc @ahrtr @ivanvc

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.5.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.15 (TBD)
 
+### etcd server
+- Fix [add prometheus metric registration for metric `etcd_disk_wal_write_duration_seconds`](https://github.com/etcd-io/etcd/pull/18164).
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-3.5.md:10]: Link should be pointing to the `release-3.5` backport pr: https://github.com/etcd-io/etcd/pull/18174

**Review by jmhbnz — COMMENTED** [review]
Thanks @andyxning - Just need to update the link then looks good to me.

File: CHANGELOG/CHANGELOG-3.5.md (hunk 1 above)
Comment (andyxning) [reply CHANGELOG/CHANGELOG-3.5.md]: > Link should be pointing to the `release-3.5` backport pr: #18174

@jmhbnz Done

**Review by jmhbnz — APPROVED** [review]
LGTM

**Review by ivanvc — APPROVED** [review]
LGTM. Thanks, @andyxning

**Review by ahrtr — APPROVED** [review]

**Pull request merged**